step-1-getting-started.md

```diff
@@ -11,30 +11,30 @@ Testcontainers libraries are compatible with Java 8+, but this workshop uses a Q
 
 We would recommend using [SDKMAN](https://sdkman.io/) to install Java on your machine if you are using MacOS, Linux or Windows WSL.
 
-### Install Docker
+### Install a container runtime
 
-You need to have a Docker environment to use Testcontainers.
+You need to have a [Docker](https://docs.docker.com/get-docker/) or [Podman](https://podman.io/) environment to use Testcontainers.
```

Comment:
I heard that Podman with Testcontainers was a bit clunky, but I never tested it though... Do you have experience with that?

Reply:
We have a whole guide dedicated to it (https://quarkus.io/guides/podman). I'll add that to the docs in another PR.